lib/rake_github/exceptions/no_pull_request_error.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module RakeGithub
+  module Exceptions
+    class NoPullRequestError < StandardError
```

Comment:
The way this exception was defined before pollutes the top level namespace which is a bad practice so I've moved it down into an `Exceptions` module within `RakeGithub`.